cmd/test.go

```diff
@@ -208,6 +208,11 @@ func opaTest(args []string) int {
 		return 1
 	}
 
+	if testParams.threshold < 0 || testParams.threshold > 100 {
```

Comment:
[nit] I think we could move this check higher up. Since we're going to end execution anyways if it fails, we can avoid the work done before.

Reply:
@hzliangbin can you PTAL? :)